src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -204,6 +205,9 @@ bool ProbeVisitor::VisitBinaryOperator(BinaryOperator *E) {
         }
       }
     }
+  } else if (IsContextMemberExpr(E->getRHS())) {
+    ProbeSetter setter(&ptregs_);
+    setter.TraverseStmt(E->getLHS());
```

Comment:
Could you have an example when this piece of code is executed at runtime?

Reply:
The second example in `test_clang.py` (kprobe example) goes through that path.
```c
// Detected in VisitVarDecl:
struct sock *sk = (struct sock *)args->skaddr;

// Detected in VisitBinaryOp:
struct sock *sk;
sk = (struct sock *)ctx->di;
```

Reply:
Right, this is an assignment binary and the right is a ctx access!